tic_tac_toe/game.py

```diff
@@ -47,10 +74,54 @@ def _is_winning_combination(board, combination, player):
     Returns True of all three positions in the combination belongs to given
     player, False otherwise.
     """
-    pass
 
 
-def _check_winning_combinations(board, player):
+def _check_winning_combinations(board, player="O"):#doit
```

Comment:
Not sure why there are two def statement lines....also is there a reason why you have default value for player?
I suggest calling _is_winning_combination defined above in this method.